templates/components/common/navigation-menu.html

```diff
@@ -210,4 +210,5 @@
             </li>
         {{/if}}
     </ul>
+    {{> components/search/quick-results-aria-message }}
```

Comment:
Could we move this to quick-search file?

Reply:
When input in focus, browser determines its wrapper as group, and reading all its content including search notification text